src/css/components/_multiple-choice.scss

```diff
@@ -0,0 +1,84 @@
+.cmp-multiple-choice {
+	display: flex;
+	gap: 2rem;
+	align-items: stretch;
+	margin-block: 2rem;
+	flex-wrap: wrap;
+
+	&__item {
+		flex: 1 1 calc(33% - 1.33rem);
+		position: relative;
+		display: flex;
+		align-items: stretch;
+	}
+
+	&__button {
+		position: absolute;
+		inset: 0;
+		z-index: 200;
+		background-color: transparent;
+		color: inherit;
+		border: none;
+		margin: 0;
+		padding: 0;
+		text-align: inherit;
+		font: inherit;
+		border-radius: 0;
+		appearance: none;
+		cursor: pointer;
```

Comment:
I'm not entirely sure why, but in Firefox, the buttons are getting a width of 0. It seems like setting `width: 100%` fixes it without affecting anything.